packages/core/src/components/selectable/selectable.scss

```diff
@@ -10,6 +10,8 @@
   &[invalid]:not([invalid="false"]) {
     @include selectable.invalid();
   }
+
+  @include selectable.children("dso-info");
```

Comment:
Is deze niet dubbelop? In `packages/dso-toolkit/src/components/selectable/selectable.scss` staat op r. 18 e.v.:

```scss
// Web Component
dso-selectable {
  @include selectable.children("dso-info"); <==?

  .dso-rich-content[slot="info"] {
    @include info.richContent();
  }
}
```

Reply:
Daar werkt het niet, althans niet waar het voor deze zou moeten werken.

Reply:
Eigenlijk zou de styling die Eric aanhaalt in z'n geheel moeten verplaatsen naar `packages/core/src/components/info/info.scss` met `::slotted` ed.

Maar de Selectable/Info constellatie is niet bepaald makkelijk te doorgronden 🤔

Reply:
Ik denk dat we het even moeten laten en zo moeten mergen. Als het er goed uitziet en nergens anders omvalt. De opzet van deze twee componenten is onder de maat en moeten we 'n keer naar kijken hoe we dit goed gaan opzetten.

@iterox als jij geen bezwaren hebt, wil je dit dan mergen?